zmon-controller-app/src/main/java/org/zalando/zmon/api/EntityApi.java

```diff
@@ -137,6 +144,16 @@ public int deleteEntity(@PathVariable(value = "id") String id) {
         List<String> deleted = entitySprocs.deleteEntity(id, teams, authService.getUserName());
 
         if (!deleted.isEmpty()) {
+            try {
+                JsonNode e = mapper.readValue(deleted.get(0), JsonNode.class);
```

Comment:
Am I right, that the type of the first deleted metric in the list is the same as for all others? Or we always have only just one metric in the list?

Reply:
there's always just one